packages/eyeglass/src/util/strings.ts

```diff
@@ -1,28 +1,95 @@
-"use strict";
+import { SassImplementation, isSassString, SassString, SassValue, toString } from "./SassImplementation";
+import { inspect } from "util";
 
-var rUnquote = /^("|')(.*)\1$/;
-var rPlaceholders = /\${([^}]+)}/g;
+let rUnquote = /^("|')(.*)\1$/;
+let rPlaceholders = /\${([^}]+)}/g;
 
-function isSassValue(item) {
-  return !!(item && item.getValue && item.constructor);
+export function unquote(sass: SassImplementation, string: string | SassValue): SassString {
```

Comment:
might be worth having a local type alias kind of like

```ts
type SassValueLike = string | SassValue;
```

Reply:
I just really hate this module's public API, tbh. I almost refactored it, but made myself stick to my rule for not making unnecessary code changes during this PR. My preference would be to have APIs that clearly take Sass values and return Sass values or take JS values and return JS values.